docker-compose.yml

```diff
@@ -32,4 +32,4 @@ volumes:
   db_data:
 
 networks:
-  app_network:
\ No newline at end of file
+  app_network:
```

Comment:
Is this just space?

Reply:
I added an end of line, it was an end of file there